index.md

```diff
@@ -9,3 +9,4 @@ Slides
 - [Getting Started on Codewars](/slides/codewars-intro.html)
 - [Tech for Better Discovery Workshop](/slides/tfb-discovery-workshop.html)
 - [Slides for non-FAC people to showcase FAC (help me with name)](slides/arsenal-slides.html)
+- [Founders programme intro for students](/slides/founders-programme)
```

Comment:
```suggestion
- [Founders programme intro for students](/slides/founders-programme.html)
```
missed out the extension. I think you can hit commit suggestion and it'll go straight in

Reply:
@sofer

Reply:
@charlielafosse It shouldn't need it. See e.g. http://facresources.com/slides/tfb

Reply:
Oh yeah, my bad